### PR TITLE
JIRA and Github issues explanation

### LINKED ISSUE #8271: Add Jira/Github Issue to "CONTIRUBTE.rst"
**What you expected to happen**:

I would be happy if Jira and Github were added to the "How to communicate" section of the CONTRIBUTING.rst file. They should present Jira's previous and current role, and describe the Github Issue and the differences between the two services.
https://github.com/apache/airflow/blob/master/CONTRIBUTING.rst#how-to-communicate

We need this description to answer questions similar to the following:
https://lists.apache.org/thread.html/rd52ec6081dd612e51e77f119ce94984cf8cbe8629da497496b7cde80%40%3Cdev.airflow.apache.org%3E

Best regards,

### PR DESCRIPTION
**Pull request by mschickensoup**
fix https://github.com/apache/airflow/issues/8271

Make sure to mark the boxes below before creating PR: [x]

- [ x] Description above provides context of the change
- [ x] Unit tests coverage for changes (not needed for documentation changes)
- [ x] Target Github ISSUE in description if exists
- [ x] Commits follow "[How to write a good git commit message](http://chris.beams.io/posts/git-commit/)"
- [ x] Relevant documentation is updated including usage instructions.
- [ x] I will engage committers as explained in [Contribution Workflow Example](https://github.com/apache/airflow/blob/master/CONTRIBUTING.rst#contribution-workflow-example).

---
In case of fundamental code change, Airflow Improvement Proposal ([AIP](https://cwiki.apache.org/confluence/display/AIRFLOW/Airflow+Improvements+Proposals)) is needed.
In case of a new dependency, check compliance with the [ASF 3rd Party License Policy](https://www.apache.org/legal/resolved.html#category-x).
In case of backwards incompatible changes please leave a note in [UPDATING.md](https://github.com/apache/airflow/blob/master/UPDATING.md).
Read the [Pull Request Guidelines](https://github.com/apache/airflow/blob/master/CONTRIBUTING.rst#pull-request-guidelines) for more information.

### CODE REVIEW

File: CONTRIBUTING.rst [hunk 1]
```diff
@@ -879,7 +879,7 @@ Apache Airflow is a Community within Apache Software Foundation. As the motto of
 the Apache Software Foundation states "Community over Code" - people in the
 community are far more important than their contribution.
 
-This means that communication plays a big role in it, and this chapter is all about it.
+This means that communication plays a big role in it, and this chapter is all about it. This is really important.
```
Comment (mik-laj) [inline CONTRIBUTING.rst]:
```suggestion
This means that communication plays a big role in it, and this chapter is all about it. This is really important.
```

File: CONTRIBUTING.rst [hunk 2]
```diff
@@ -899,6 +899,13 @@ You can join the channels via links at the `Airflow Community page <https://airf
 * Github `Pull Requests (PRs) <https://github.com/apache/airflow/pulls>`_ for:
    * discussing implementation details of PRs
    * not for architectural discussions (use the devlist for that)
+* The deprecated `JIRA issues <https://issues.apache.org/jira/projects/AIRFLOW/issues/AIRFLOW-4470?filter=allopenissues>`_ for:
+   * checking out old but still valuable issues that are not on Github yet
+   * mentioning the JIRA issue number in the title of the related PR you would like to open on Github
+   
+**IMPORTANT**
```
Comment (mik-laj) [inline CONTRIBUTING.rst]:
```suggestion
**IMPORTANT**
```

File: CONTRIBUTING.rst [hunk 3]
```diff
@@ -899,6 +899,13 @@ You can join the channels via links at the `Airflow Community page <https://airf
 * Github `Pull Requests (PRs) <https://github.com/apache/airflow/pulls>`_ for:
    * discussing implementation details of PRs
    * not for architectural discussions (use the devlist for that)
+* The deprecated `JIRA issues <https://issues.apache.org/jira/projects/AIRFLOW/issues/AIRFLOW-4470?filter=allopenissues>`_ for:
+   * checking out old but still valuable issues that are not on Github yet
+   * mentioning the JIRA issue number in the title of the related PR you would like to open on Github
+   
+**IMPORTANT** 
+We don't create new issues on JIRA anymore. The reason we still look at JIRA issues is that there are valuable tickets inside of it. However, each new PR should be created on _`Github issues <https://github.com/apache/airflow/issues>_` as stated in _`Contribution Workflow Example <https://github.com/apache/airflow/blob/master/CONTRIBUTING.rst#contribution-workflow-example>`_
```
Comment (mik-laj) [inline CONTRIBUTING.rst]:
```suggestion
We don't create new issues on JIRA anymore. The reason we still look at JIRA issues is that there are valuable tickets inside of it. However, each new PR should be created on _`Github issues <https://github.com/apache/airflow/issues>_` as stated in `Contribution Workflow Example <https://github.com/apache/airflow/blob/master/CONTRIBUTING.rst#contribution-workflow-example>`_
```

File: CONTRIBUTING.rst [hunk 4]
```diff
@@ -879,7 +879,7 @@ Apache Airflow is a Community within Apache Software Foundation. As the motto of
 the Apache Software Foundation states "Community over Code" - people in the
 community are far more important than their contribution.
 
-This means that communication plays a big role in it, and this chapter is all about it.
+This means that communication plays a big role in it, and this chapter is all about it. This is really important.
```
Comment (mik-laj) [inline CONTRIBUTING.rst]:
```suggestion
This means that communication plays a big role in it, and this chapter is all about it. 
```

Comment (mik-laj) [reply CONTRIBUTING.rst]: This sentence is strong enough. WDYT? In addition, it is in a separate paragraph.

File: CONTRIBUTING.rst [hunk 5]
```diff
@@ -899,6 +899,13 @@ You can join the channels via links at the `Airflow Community page <https://airf
 * Github `Pull Requests (PRs) <https://github.com/apache/airflow/pulls>`_ for:
    * discussing implementation details of PRs
    * not for architectural discussions (use the devlist for that)
+* The deprecated `JIRA issues <https://issues.apache.org/jira/projects/AIRFLOW/issues/AIRFLOW-4470?filter=allopenissues>`_ for:
+   * checking out old but still valuable issues that are not on Github yet
+   * mentioning the JIRA issue number in the title of the related PR you would like to open on Github
+   
+**IMPORTANT**
+We don't create new issues on JIRA anymore. The reason we still look at JIRA issues is that there are valuable tickets inside of it. However, each new PR should be created on _`Github issues <https://github.com/apache/airflow/issues>_` as stated in `Contribution Workflow Example <https://github.com/apache/airflow/blob/master/CONTRIBUTING.rst#contribution-workflow-example>`_
```
Comment (potiuk) [inline CONTRIBUTING.rst]:
```suggestion
We don't create new issues on JIRA anymore. The reason we still look at JIRA issues is that there are valuable tickets inside of it. However, each new PR should be created on `Github issues <https://github.com/apache/airflow/issues>`_ as stated in `Contribution Workflow Example <https://github.com/apache/airflow/blob/master/CONTRIBUTING.rst#contribution-workflow-example>`_
```

File: CONTRIBUTING.rst [hunk 6]
```diff
@@ -899,6 +899,13 @@ You can join the channels via links at the `Airflow Community page <https://airf
 * Github `Pull Requests (PRs) <https://github.com/apache/airflow/pulls>`_ for:
    * discussing implementation details of PRs
    * not for architectural discussions (use the devlist for that)
+* The deprecated `JIRA issues <https://issues.apache.org/jira/projects/AIRFLOW/issues/AIRFLOW-4470?filter=allopenissues>`_ for:
+   * checking out old but still valuable issues that are not on Github yet
+   * mentioning the JIRA issue number in the title of the related PR you would like to open on Github
+
```
Comment (mik-laj) [inline CONTRIBUTING.rst]:
```suggestion

```

**Comment by mik-laj** [comment]
@mschickensoup Do you need any help?

File: CONTRIBUTING.rst [hunk 7]
```diff
@@ -879,7 +879,7 @@ Apache Airflow is a Community within Apache Software Foundation. As the motto of
 the Apache Software Foundation states "Community over Code" - people in the
 community are far more important than their contribution.
 
-This means that communication plays a big role in it, and this chapter is all about it.
+This means that communication plays a big role in it, and this chapter is all about it.
```
Comment (mik-laj) [inline CONTRIBUTING.rst]:
```suggestion
This means that communication plays a big role in it, and this chapter is all about it.
```

**Review by potiuk — APPROVED** [review]

**Comment by potiuk** [comment]
Congrats @mschickensoup !

**Comment by mik-laj** [comment]
Fantastic work! I am waiting for your next contributions. 🐈

**Comment by mschickensoup** [comment]
Awwww :3 Thank you guys! @mik-laj @potiuk

**Comment by mik-laj** [comment]
@mschickensoup Do you have ideas for your next contribution? What would you like to do in this project?

**Pull request merged**